tic-tac-toe.js

```diff
@@ -1,188 +1,172 @@
-let players = ["player1", "player2"];
-let states = ["open", "player1won", "player2won", "draw"];
+const Marks = {
+  P1: "X",
+  P2: "O",
+  Empty: ".",
+};
+
+const players = {
+  P1: "player1",
+  P2: "player2",
+};
+
+const states = {
+  open: "open",
+  P1: "player1won",
+  P2: "player2won",
+  dran: "draw",
+};
+
+/*const players = ["player1", "player2"];
```

Comment:
Usually, old code has to be deleted.

If it is something that you want to be able to access in the future you can make a commit message that explains why it was deleted/changed

Reply:
Oks. I'll delete it.